<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCMapp</class>
 <widget class="QMainWindow" name="DCMapp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCMapp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="GetArg">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Options</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="argumentArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>761</width>
      <height>411</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>759</width>
       <height>409</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="executeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outputConsole">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>536</y>
      <width>621</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="exes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
